repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -55,6 +57,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
     }
   }
 
+  override def getResource(name: String): URL = {
+    parentLoader.getResource(name)
```

Comment:
@vanzin 
I didn't see use case you mentioned but it could make sense.

In order to achieve, we have to implement findResource() and findResources() for ExecutorClassLoader since ExecutorClassLoader cannot rely on superclass (ClassLoader) to load class / resource.
It is easy to provide resource URL which points to origin scheme (http, https, ftp, hdfs), but since I'm new to classloader, so I'm wondering it is safe to return URL from findResource() and findResources() which doesn't point to local file.

If it is not safe to return non local file as URL, what's recommended way to do?
I can only think about downloading files to local temp directory per every call.